prescreeners/va.html

```diff
@@ -273,7 +273,7 @@ <h3 class="usa-alert__heading">Information for immigrant households</h3>
                    value="Submit">
           </form>
           <div id="errors"></div>
-          <div id="results"></div>
+          <div id="results" role="alert"></div>
```

Comment:
Tell screenreaders like VoiceOver to read the results immediately after they are received; otherwise the user won't be informed that they've received results after submitting the form.